### DISCUSSION TITLE
Workflow

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#982] [Discussion]
Who here uses github workflow?

**Comment by ghost** [comment]
So..no one?

**Comment by apple502j** [comment]
It doesn't seem like this is related to Scratch Addons. closing.

**Reply by Hans5958** [reply]
It is related... to the development?

**Reply by ghost** [reply]
> It doesn't seem like this is related to Scratch Addons. closing.

It wasn't supposed to be. Where else would I put it? The discussion category description literally says: stuff that doesn't belong anywhere else.

**Comment by Hans5958** [comment]
I do. What's the matter?

**Reply by ghost** [reply]
Just wanted to know.
